### DISCUSSION TITLE
The project status

### DISCUSSION THREAD

**Discussion by AliOsm** [ruby-ui/ruby_ui#88] [General]
Hello everyone, and thanks for
 your great contributions.

I want to understand the status of the project, what are the plans for the rest of components, and the v1 release. We are planning to adapt Phlex and PhlexUI in multiple projects, and if the pro version released, we are planning to subscribe also. So, any ideas on the previous asks?

**Comment by cirdes** [comment]
Hey @AliOsm ,

@iseth and I are working on the next version of PhlexUI. Let me pinpoint what the priorities are:

- We are working on the V1 branch, and the main branch has been frozen.
- We are renaming the project to RBUI (Ruby UI). We want to support helpers and ViewComponent wrappers so you can use the component library any way you want.
- We are creating generators so you can eject a component and customize it.
- The new website rbui.dev is available, and you can see the new components there.
- We added two important new components: Combobox and Forms.
- We are focusing on refining the current components instead of adding new ones.
- The pro version has been merged into the free one. The component library is now 100% free.

Let me know if you need anything else.

**Reply by AliOsm** [reply]
Thanks for the updates, this is very exciting tbh, as I already migrated ~30% of my project's views into Phlex/PhlexUI, and working on another project.

Based of your reply, is it safe to use v1 branch in my gemfile? I really like the Chart component 🤩

Also, any good starter issue that I can help with? I'm really excited about the project 😅

**Reply by sethhorsley** [reply]
It would be super cool to add a drawer like:
https://ui.shadcn.com/docs/components/drawer

**Comment by BarnabeD** [comment]
Hello, 
Thank you @AliOsm for your question, seeing some PR merged, I asked myself the same question.

Thank you @cirdes and @iseth for the clear vision and all this information.
Phlex UI help me every day to speed up my Phlex project. I love it.
